src/loaders/eventLoader.js

```diff
@@ -96,11 +96,12 @@ const loadEventLayer = async config => {
     // Delete serverCluster option if previously set
     delete config.serverCluster;
 
-    if (spatialSupport && eventClustering) {
+    // Check if events should be clustered on the server or the client
+    // Style by data item is only supported in the client (donuts)
+    if (spatialSupport && eventClustering && !styleDataItem) {
         const response = await getCount(analyticsRequest);
         config.bounds = getBounds(response.extent);
```

Comment:
`config.bounds` won't be set now when `styleDataItem` is true, as it used to be before, but maybe this is irrelevant for this case 🤷‍♂️

Reply:
yes, see note about it in the description above :-)

Reply:
> "config.bounds" is only needed for server cluster when we zoom the map to the combined bounds of all the events.
nvm, clearly I need to read the description better 😄